sdk/azurestackhci/Azure.ResourceManager.Hci/src/Generated/Models/SkuMappings.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Hci.Models
+{
+    /// <summary> SKU Mapping details. </summary>
+    public partial class SkuMappings
+    {
+        /// <summary> Initializes a new instance of SkuMappings. </summary>
+        public SkuMappings()
+        {
+            MarketplaceSkuVersions = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Initializes a new instance of SkuMappings. </summary>
+        /// <param name="catalogPlanId"> Identifier of the CatalogPlan for the sku. </param>
+        /// <param name="marketplaceSkuId"> Identifier for the sku. </param>
+        /// <param name="marketplaceSkuVersions"> Array of SKU versions available. </param>
+        internal SkuMappings(string catalogPlanId, string marketplaceSkuId, IList<string> marketplaceSkuVersions)
+        {
+            CatalogPlanId = catalogPlanId;
+            MarketplaceSkuId = marketplaceSkuId;
+            MarketplaceSkuVersions = marketplaceSkuVersions;
+        }
+
+        /// <summary> Identifier of the CatalogPlan for the sku. </summary>
+        public string CatalogPlanId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
Changed